app/cdap/components/Connections/index.tsx

```diff
@@ -73,6 +73,7 @@ export default function Connections({
   initPath,
   connectorType,
   hideAddConnection,
+  allowDefaultConnection = true,
```

Comment:
This has a default because my editor yelled at me in a way I didn't feel like debugging when I tried to use `render={() => <Connections allowDefaultConnection={true}/>}` in the `<Route/>` in https://github.com/cdapio/cdap-ui/blob/ee5f2e1b040c099394586b76df707557afe6d00c/app/cdap/components/Home/index.js#L150 - however, I think it might make more sense to make that latter change. Reviewer, please advise :)

Reply:
Could you set the default in `ConnectionsContext`? That might be a little cleaner. But I think setting it this way, not in the `Route`, is the way to go.